skins/oasis/css/core/ads-porvata.scss

```diff
@@ -0,0 +1,282 @@
+@import '../../../../extensions/wikia/ArticleVideo/styles/video-shared-variables';
+
+$wikia-top-ads-index: 2;
+
+$default-progress-bar-color: #b7b7b7;
```

Comment:
can we use DS colors? http://wikiadesignsystem.com/#/base-styles/colors
They're slightly different but I think it's more important to have the same color palette.

Reply:
This one is used for VUAP and I wouldn't like to change it. I can use new values for outstream, is it fine?